src/flutter_ffi.rs

```diff
@@ -751,6 +751,25 @@ pub fn main_load_recent_peers() {
     }
 }
 
+pub fn main_load_recent_peers_sync() -> SyncReturn<String> {
+    if !config::APP_DIR.read().unwrap().is_empty() {
+        let peers: Vec<HashMap<&str, String>> = PeerConfig::peers()
+            .drain(..)
+            .map(|(id, _, p)| peer_to_map(id, p))
+            .collect();
+
+        let data = HashMap::from([
+            ("name", "load_recent_peers".to_owned()),
+            (
+                "peers",
+                serde_json::ser::to_string(&peers).unwrap_or("".to_owned()),
+            ),
+        ]);
+        return SyncReturn(serde_json::ser::to_string(&data).unwrap_or("".to_owned()));
+    }
+    SyncReturn("".to_string())
+}
+
```

Comment:
If added this, main_load_recent_peers can be removed, no need two.

Reply:
```
         let _res = flutter::push_global_event(
             flutter::APP_TYPE_MAIN,
             serde_json::ser::to_string(&data).unwrap_or("".to_owned()),
         );
```
 looks ugly.

Reply:
current the recent peers, etc are loading with this global events by @fufesou . Maybe we need to change the `PeerCard` from events driven to direct-call driven @fufesou ?

Reply:
I think direct replacement is problematic.
Because there are some places where main_load_recent_peers is used as a trigger event to re-render the ui.